oci/runtime_v2.go

```diff
@@ -192,6 +192,15 @@ func (r *RuntimeV2) startRuntimeDaemon(c *Container) error {
 
 // StartContainer starts a container.
 func (r *RuntimeV2) StartContainer(c *Container) error {
+	logrus.Debug("Runtimev2.StartContainer() start")
+	defer logrus.Debug("RuntimeV2.StartContainer() end")
+
+	if _, err := r.task.Start(r.ctx, &task.StartRequest{
+		ID: c.ID(),
+	}); err != nil {
```

Comment:
If start failed, should we call task delete to do some cleanup?

Reply:
Definitely.

Reply:
Actually, after I put more thoughts into this one, I think we should not call into delete since this code should only rollback what has been done into this function. Upper layer such as `server/container_start.go` should be responsible of deleting the container if it wants to.